### ISSUE TITLE
How to predict after finetune? There is no config.json in checkpoint dir.

### ISSUE THREAD

**Issue by lyt719** [huggingface/parler-tts#68]

**Comment by lyt719** [comment]
> Hey @lyt719 - could you share reproducible launch scripts you used for fine-tuning? It would also be great to have a snapshot of the checkpoint dir to confirm that it's just the config.json that's missing.
> 
> Note that the config.json file isn't changed during training. Only the weights of the model are changed (). Thus, you should be able to copy the original config into your checkpoint dir and subsequently load the weights.`model.safetensors`

i got optimizer.bin, pytorch_model.bin, random_states_0.pkl, scheduler.bin four files in checkpoint dir, I have copied all json from parler_tts_mini_v0.1 to my checkpoint dir. ParlerTTSForConditionalGeneration.from_pretrained is OK. But when i run AutoTokenizer.from_pretrained there is an error.
![Uploading 屏幕截图 2024-06-17 094343.png…]()
May i ask how to write the path in AutoTokenizer.from_pretrained() in predict after finetune? Is it my checkpoint dir?

**Comment by lyt719** [comment]
![屏幕截图 2024-06-17 094343](https://github.com/huggingface/parler-tts/assets/58416622/788fcffa-4f7a-48bc-a54b-a3d886b2e9d9)
